t/14-grutasks.t

```diff
@@ -601,6 +602,27 @@ subtest 'download assets with correct permissions' => sub {
     combined_like { run_gru_job($t->app, 'download_asset' => [$assetsource, $assetpath, 0]) }
     qr/Skipping download of "$assetsource" to "$assetpath" because file already exists/, 'everything logged';
     ok -f $assetpath, 'asset downloaded';
+
+    my $cwd          = getcwd;
+    my @destinations = (
+        "$cwd/t/data/openqa/share/factory/iso/test1.iso",
+        "$cwd/t/data/openqa/share/factory/iso/test2.iso",
+        "$cwd/t/data/openqa/share/factory/iso/test3.iso"
+    );
+    combined_like { run_gru_job($t->app, 'download_asset' => [$assetsource, \@destinations, 0]) }
+    qr /Download of "$destinations[0]" successful/, 'download task successed';
+    my $destination = shift @destinations;
+    ok -f $destination, 'asset download successfully';
+    foreach (@destinations) {
+        ok -f $_, "$_ symlink successfully";
+        is readlink($_), $destination, "the source file is correct";
+    }
+    system("rm -f $destination");
```

Comment:
Did `path($destination)->remove` not work here? The same counts for the next occurrence.